<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addEvents</class>
 <widget class="QMainWindow" name="addEvents">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a new EVENT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Start Date</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Reminder Time</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Event Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="10" column="0" colspan="5">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Add Event</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Start Time</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Place</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>End Date</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="QLineEdit" name="lineEdit_7"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>End Time</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>723</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
